lib/kuikka_website/web/templates/layout/sidebar.html.eex

```diff
@@ -5,7 +5,20 @@
     <img src="/images/logo.svg" />
   </div>
   <nav class="g--12">
-    <a href="<%= forum_path(@conn, :index) %>">Forums</a>
+    <%= if topics = forum_topics()  do %>
+      <div class="nav-collapsible">
```

Comment:
They are there just for testing. After topics starts showing I will move them to seperate page.